docs: Add docs for SSH Keys and Cloud Provider Credentials (#541)
devopness/devopness: docs/cloud-providers/add-cloud-providers-credentials.md

---
title: Add a Cloud Provider Credential
intro: Devopness allows you to manage multiple Cloud Providers accounts from a single platform.
Add Cloud Providers Credentials to your Devopness account to provision and manage cloud infrastructure resources with increased productivity.
links:
    overview:
    quickstart:
    previous:
    next:
    guides:
    related:
        - /docs/cloud-providers/remove-cloud-providers-credentials
    featured:
---

# Add a Cloud Provider Credential
1. In the upper-right corner of any page, click your profile icon then click `Cloud Providers`
    - A list of existing Cloud Providers Credentials will be displayed
2. On the upper-right corner of the list click `ADD CLOUD PROVIDER`
3. Use the drop-down menu to choose a Cloud Provider

> 4. Provide a name to the `Cloud Provider Credential` being added. For example, "Company Laptop"

Author: @mlpbraga thanks! what about?
```suggestion
4. Provide a name to your new `Cloud Provider Credential`. For example, "Test Account"
```
